tracker/views.py

```diff
@@ -90,44 +90,23 @@ def get(self, request):
 
     # post: retrieves the form data to save and reload the home page
     def post(self, request):
-        form = DataForm()
-        form = DataForm(request.POST)
+        form = CreateTaskForm()
+        form = CreateTaskForm(request.POST)
         
-        if form.is_valid:
+        if form.is_valid():
             #------------------------------------------------------------------------------------#
             # save form data with commit=False. If commit= False, data haven't add in database
-            # form.data is QueryDict and is not mutable. I cann't save it with less data
-            # I cann't add out-of-form data as estimateb_by_calc and correctness
             #------------------------------------------------------------------------------------#
-            form.save(commit=False)
             
-            # calculation of estimateb_by_calc and correctness from user data
-            estimated_dt = datetime.strptime(form.data["estimate"], '%H:%M')
-            realtime_dt = datetime.strptime(form.data["realtime"], '%H:%M')
+            instance =  form.save(commit=False)
 
-            estimated_duration_td = timedelta(hours=estimated_dt.hour, minutes=estimated_dt.minute)
-            realtime_duration_td = timedelta(hours=realtime_dt.hour, minutes=realtime_dt.minute)
+            # caclulate estimation and correctness
+            estimateb_by_calc=calc_estimation_time(form.data["estimate"], form.data["realtime"])
+            correctness=calc_correctness(form.data["estimate"], form.data["realtime"])
 
```

Comment:
Why are you using some temp variable here? You could do instance.estimateb_by_calc = alc_estimation_time(form.data["estimate"], form.data["realtime"]) direclty?

Reply:
Same for correctness